tests/docker-compose.yml

```diff
@@ -11,9 +11,12 @@ services:
       - "8880"
   hdfs:
     image: rkuprieiev/docker-hdfs
+    hostname: localhost
     ports:
       - "8020"
       # NOTE: having this port as dynamic one will require modifying
       # `dfs.datanode.address` in `hdfs-site.xml` and probably something
       # else, so using default one for now.
       - "50010:50010"
+      - "50075:50075"
```

Comment:
Do we need this one?

Reply:
We kind of do: when creating a file `hdfs` makes an initial `PUT` request to port `50070`. This request is answered with a temporary redirect to the datanode listening in port `50075`, not the dynamic port. So without having the port as static `hdfs` will fail when following the redirect, as it will attempt to request `50075`.